### PR TITLE
List the minor version that introduced ttlOverride

### LINKED ISSUE #7350: CDNs documentation doesn't list the minor version that introduced ttlOverride
## This Bug Report affects these Traffic Control components:
- Documentation

## Current behavior:
The `ttlOverride` property of a CDN doesn't exist prior to APIv4.1, but the docs make no mention of that which could lead a user to incorrectly believe that the property can be used in APIv4.x for any value of `x`.

## Expected behavior:
Should have a `.. versionadded::` directive indicating that it was added in APIv4.1

## Steps to reproduce:
Read the docs

### PR DESCRIPTION
**Pull request by zrhoffman**
This PR fixes #7350 by listing the minor version that introduced ttlOverride.
<hr/>

## Which Traffic Control components are affected by this PR?

- Documentation

## What is the best way to verify this PR?

Build and look at the docs

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Review by ocket8888 — APPROVED** [review]
I can merge this as-is if you want - it does fix #7350 - but note that the docs for `cdns/{{ID}}` have a similar issue; they are missing `ttlOverride` entirely, in both v4 and v5. If you don't wanna fix that here I'll just make another issue after this PR is merged.

**Comment by zrhoffman** [comment]
> I can merge this as-is if you want - it does fix #7350 - but note that the docs for `cdns/{{ID}}` have a similar issue; they are missing `ttlOverride` entirely, in both v4 and v5. If you don't wanna fix that here I'll just make another issue after this PR is merged.

nah let's fix it here

**Comment by zrhoffman** [comment]
Added for `PUT cdns/{{ID}}` in 4ee27cb0c4

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**
